10up/classifai
# Added Generate Excerpt for the Classic Editor.

### Description of the Change
Adds the Excerpt Generation (introduced in https://github.com/10up/classifai/pull/405 for Gutenberg) to the classic editor.

Closes #417

### How to test the Change

1. Create a post using the classic editor
2. Add some content and save as a draft (note: a draft must be saved as we need a post ID to make the OpenAI request)
3. Locate the excerpt box and click "Generate excerpt"
4. Button should be disabled; content should be added to the excerpt `textarea`; button text is updated (to Re-generate excerpt); button should be enabled.

### Changelog Entry
> Added - Excerpt Generation support in the classic editor.

### Credits
Props @jamesmorrison, @ravinderk, @dkotter 


### Checklist:
